checkipa

```diff
@@ -255,7 +275,10 @@ class IntegrityCheck(object):
                 txt.append('  %s' % (udid))
         else:
             txt.append('  Provisioned Devices (0):')
-
+        txt.append('  Package size: %2.2f GB' % (self.ipa_size/1000000000.0))
+        txt.append('  Executable size: %2.2f MB' % (self.executable_size/1000000.0))
+        txt.append('  MD5: %s ' % self.ipa_md5)
+        
```

Comment:
The above 3 lines (278, 279, and 280) do not check validity of the variables self.ipa_size, self.executable_size, and self.ipa_md5 before they are used